db/migrations/20171113141257_create_wods.cr

```diff
@@ -0,0 +1,14 @@
+class CreateWods::V20171113141257 < LuckyMigrator::Migration::V1
+  def migrate
+    create :wods do
+     add description : String
+     add date : Time
+    end
+
+    execute "CREATE UNIQUE INDEX wods_date_index ON wods (date);"
```

Comment:
This will be a lot simpler once this merges  https://github.com/luckyframework/migrator/pull/17 Just FYI :D